app/models/post.rb

```diff
@@ -4,4 +4,27 @@ class Post < ActiveRecord::Base
 	# ------------------------------------------------------------------------------------------------------
 	belongs_to :user, counter_cache: true
 
+
+  # LINE WITH 100 CHARACTERS
+  # ------------------------------------------------------------------------------------------------
+
+  # LINE WITH MORE THAN 100 CHARACTERS
+  # ------------------------------------------------------------------------------------------------------
+
+  # LINE WITH LESS THAN 100 CHARACTERS
+  # -----------------------------------------------------------------------------------
```

Comment:
Line is too long. [87/80]<br>Incorrect indentation detected (column 2 instead of 1).